### PR TITLE
build: add react storybook

### PR DESCRIPTION
**Pull request by AliKdhim87**

### CODE REVIEW

File: packages/storybook-react/config/preview.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import '@utrecht/component-library-vue/dist/style.css';
```
Comment (Robbert) [inline packages/storybook-react/config/preview.js:1]: Is this `component-library-vue` import correct, or should it be `component-library-css`? In any case the correct dependency should also be added to `package.json`.

File: packages/storybook-react/package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "@utrecht/storybook": "file:../storybook"
+  },
+  "devDependencies": {
+    "@babel/core": "7.18.10",
+    "@storybook/addon-actions": "6.5.10",
+    "@storybook/addon-essentials": "6.5.10",
+    "@storybook/addon-interactions": "6.5.10",
+    "@storybook/addon-links": "6.5.10",
+    "@storybook/builder-webpack5": "6.5.10",
+    "@storybook/manager-webpack5": "6.5.10",
```
Comment (Robbert) [inline packages/storybook-react/package.json:36]: Aren't all other packages still on `6.5.9`?

File: packages/storybook/config/main.js [hunk 3]
```diff
@@ -33,13 +33,21 @@ module.exports = {
           title: 'Vue.js Components',
           url: 'http://localhost:6007',
         },
+        react: {
+          title: 'Vue.js Components',
```
Comment (Robbert) [inline packages/storybook/config/main.js:37]: `Vue.js` → `React`

File: packages/storybook/config/main.js [hunk 4]
```diff
@@ -33,13 +33,21 @@ module.exports = {
...
+        },
       };
     }
     return {
       vue: {
         title: 'Vue.js Components',
         url: '../storybook-vue',
       },
+      react: {
+        title: 'Vue.js Components',
```
Comment (Robbert) [inline packages/storybook/config/main.js:48]: Vue.js → React

File: packages/storybook-react/config/main.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+/* eslint-env node */
+module.exports = {
+  stories: ['../src/stories/**/*.stories.mdx', '../src/stories/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials', '@storybook/addon-interactions'],
```
Comment (Robbert) [inline packages/storybook-react/config/main.js:4]: Still need to add those other addons.

File: packages/storybook-react/package.json [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  "private": true,
+  "repository": {
+    "type": "git+ssh",
+    "url": "git@github.com:nl-design-system/utrecht.git"
+  },
+  "scripts": {
+    "build": "build-storybook --output-dir dist/ --config-dir config/ ",
+    "postbuild": "npx storybook extract dist",
+    "clean": "rimraf dist/",
+    "start": "npx http-server dist/",
```
Comment (Robbert) [inline packages/storybook-react/package.json:19]: `npx` should never be necessary inside npm scripts, only outside `package.json`

**Review by Robbert — COMMENTED** [review]
Will spend some more time later and test this branch.

File: packages/storybook-react/config/preview.js (hunk 1 above)
Comment (AliKdhim87) [reply packages/storybook-react/config/preview.js:1]: This `component-library-css` should be the correct one

**Review by Robbert — APPROVED** [review]

**Pull request merged**